src/AccessibilityInsights/Modes/TestModeControl.xaml

```diff
@@ -58,9 +58,9 @@
             </Grid.RowDefinitions>
             <TextBlock Grid.Row="0" x:Name="tbSelectElement" Style="{StaticResource TbFocusableCenter}" MinWidth="40">
                 <Run Text="{x:Static properties:Resources.TestModeControl_RunAutomatedChecks}"/>
-                <fabric:FabricIconControl GlyphName="SearchAndApps" GlyphSize="Custom" FontSize="{DynamicResource StandardTextSize}" Margin="0,-3" Foreground="{DynamicResource ResourceKey=IconBrush}" FontStyle="Normal"/>
+                <fabric:FabricIconControl GlyphName="SearchAndApps" GlyphSize="Custom" FontSize="{DynamicResource StandardTextSize}" Margin="0,-3" Foreground="{DynamicResource ResourceKey=IconBrush}" FontStyle="Normal" ShowInControlView="False"/>
```

Comment:
![icons used in instructions of automated checks](https://user-images.githubusercontent.com/7775527/64989482-7746fd80-d882-11e9-9ee0-bb3ac9db6848.png)